net-dns/knot/files/knot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Knot high-performance DNS Server
+After=network.target
+
+[Service]
+ExecStart=/usr/sbin/knotd
+ExecReload=/usr/sbin/knotc reload
+ExecStop=/usr/sbin/knotc stop
+PrivateTmp=true
+RuntimeDirectory=knot
```

Comment:
Necessary as knot will not create this directory on its own, this closely mirrors the behavior of knot.init